python-sdk/example_dags/example_mssql_transform.py

```diff
@@ -56,24 +63,26 @@ def union_table_and_dataframe(input_table: Table, input_dataframe: pd.DataFrame)
     schedule_interval=None,
     start_date=START_DATE,
     catchup=False,
+    is_paused_upon_creation=False,
+    default_args=default_args,
 ) as dag:
     imdb_movies = aql.load_file(
         input_file=File(path="s3://astro-sdk/imdb_v2.csv"),
         task_id="load_csv",
-        output_table=Table(conn_id="mssql_conn"),
+        output_table=Table(conn_id="mysql_conn"),
```

Comment:
Let's define a constant on the top of the file so we avoid hard-code constants in the middle of the DAG

Reply:
resolved